policy-engine/src/main.rs

```diff
@@ -83,6 +87,9 @@ fn main() -> Result<(), Error> {
     .bind((settings.status_address, settings.status_port))?
     .run();
 
+    // Enable tracing
+    init_tracer("policy-engine")?;
```

Comment:
Make this configurable?

Reply:
Referring to the equivalent thread in the graph-builder: https://github.com/openshift/cincinnati/pull/266/files#r417288000